source/server/config/http/ip_tagging.cc

```diff
@@ -0,0 +1,36 @@
+#include "server/config/http/ip_tagging.h"
+
+#include <string>
+
+#include "common/http/filter/ip_tagging_filter.h"
+#include "common/json/config_schemas.h"
+
+namespace Envoy {
```

Comment:
You can still write tests for loading the config under https://github.com/lyft/envoy/blob/master/test/server/config/http/config_test.cc